package.json

```diff
@@ -1,1699 +1,1699 @@
-            "patterns": [
-                "src"
-            ]
-        }
-    },
-    "nyc": {
-        "include": [
-            "src/**/!(*.spec).ts"
-        ],
-        "extension": [
-            ".ts"
-        ],
-        "require": [
-            "ts-node/register",
-            "source-map-support/register"
-        ],
-        "reporter": [
-            "text-summary",
-            "html"
-        ],
-        "sourceMap": true,
-        "instrument": true,
-        "check-coverage": true,
-        "lines": 5,
-        "statements": 5,
-        "functions": 5,
-        "branches": 5
-    },
-    "categories": [
-        "Formatters",
-        "Programming Languages",
-        "Debuggers",
-        "Snippets"
-    ]
-}
+{
+    "name": "brightscript",
+    "displayName": "BrightScript Language",
+    "version": "2.18.0",
+    "publisher": "RokuCommunity",
+    "description": "Language support for Roku's BrightScript language.",
+    "author": {
+        "name": "Bronley Plumb",
+        "email": "bronley@gmail.com"
+    },
+    "icon": "images/icon.png",
+    "license": "MIT",
+    "keywords": [
+        "multi-root-ready"
+    ],
+    "engines": {
+        "vscode": "^1.30.0",
+        "node": "^7.9.0"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/RokuCommunity/vscode-brightscript-language"
+    },
+    "bugs": {
+        "url": "https://github.com/RokuCommunity/vscode-brightscript-language/issues"
+    },
+    "scripts": {
+        "preversion": "npm run build && npm run lint && npm run test && npm run check-extraneous && node scripts/preversion.js",
+        "vscode:prepublish": "npm run build",
+        "copy-schema": "cpx ./node_modules/brighterscript/bsconfig.schema.json ./dist -L",
+        "build": "npm run copy-schema && tsc -p ./ && npm run svelte-build",
+        "lint": "tslint \"./src/**/*.ts\"",
+        "watch": "npm run copy-schema && tsc -w -p ./",
+        "test": "nyc mocha \"src/**/*.spec.ts\" \"ui/**/*.spec.ts\" --full-trace --bail",
+        "test:nocover": "ts-mocha \"src/**/*.spec.ts\" \"ui/**/*.spec.ts\" --full-trace",
+        "publish-coverage": "nyc report --reporter=text-lcov | coveralls",
+        "check-extraneous": "npm list --production --parseable --depth=99999",
+        "package": "vsce package",
+        "package-local": "node scripts/package-local.js",
+        "publish": "vsce publish",
+        "clean": "rimraf dist",
+        "install-local": "node scripts/install-local.js",
+        "install-pull-local": "node scripts/install-local.js --pull",
+        "uninstall-local": "node scripts/uninstall-local.js",
+        "validate": "svelte-check",
+        "svelte-build": "rollup -c",
+        "svelte-watch": "rollup -c -w"
+    },
+    "dependencies": {
+        "array-sort": "^1.0.0",
+        "backoff": "^2.5.0",
+        "brighterscript": "^0.37.1",
+        "brighterscript-formatter": "^1.6.0",
+        "chokidar": "^3.5.1",
```

Comment:
Should this be a devDependency? Was this to help with reloading during svelte development?

Reply:
It's only designed to be used during development to allow hot reloading of webviews built with Svelte. I did it as devDependency to make the version distributed a little smaller. Can switch it a dependency if you think we should. Was double checking we were still using it as I know we'd talked about using vscode.workspace.createFileSystemWatcher but remembered that we'd tried that but it wouldn't allow since it wasn't actually part of the workspace hence the use of chokidar.

Reply:
From your comment, it feels like you believe it's already a devDependency, Based on this package.json, `chokidar` is listed as a _dependency_, and I was asking if it should be moved to a _devDependency_.